_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-resolve-redirect.php

```diff
@@ -36,7 +36,9 @@ public function register_routes() {
 						'description'       => __( 'The URL to check for redirects.', 'jetpack' ),
 						'type'              => 'string',
 						'required'          => 'true',
-						'validate_callback' => 'wp_http_validate_url',
+						'validate_callback' => function ( $param, $request, $key ) {
```

Comment:
I think we only need the first parameter in the method signature. Also, anonymous functions are supported by the minimum PHP version required by Jetpack?

Reply:
Weird I thought I had changed that! But yes anonymous functions should be fine. They were added in 5.3